src/AppKit/Enums.cs

```diff
@@ -22,34 +22,51 @@
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
 using System;
+using System.Runtime.Versioning;
```

Comment:
This file is processed by the generator:

https://github.com/xamarin/xamarin-macios/blob/697e5b330e3a1fc9dd05146ccae28ef591a1ee79/src/frameworks.sources#L79-L81

so I don't think you have to modify it?